### ISSUE TITLE
Accessibility - Borders missing in High Contrast mode for UI5 WC FlexibleColumnLayout

### ISSUE THREAD

**Issue by Teju-SFSF** [SAP/ui5-webcomponents#3320]
### **Bug Description**
Borders are missing in High Contrast mode for UI5 WC FlexibleColumnLayout near the "<" arrow.

### **Expected Behavior**
Borders should be visible in High Contrast mode for UI5 WC FlexibleColumnLayout near the "<" arrow. This is allow the data to be viewed properly divided on the page for the 3 columns.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-flexiblecolumnlayout--default-story
2. Click on Theme -> sap_fiori_3_hcb OR Theme -> sap_fiori_3_hcw
3. The three columns do not have any division (like a line, etc) to show division as in sap_fiori_3 theme.

### **Context**
- OS/Platform: Windows, Mac
- Browser: IE, Chrome
- Affected component: UI5 WC FlexibleColumnLayout

### **Log Output / Stack Trace / Screenshots**
![image](https://user-images.githubusercontent.com/84871522/119711980-fa20a980-be14-11eb-9868-bf1698b88a4a.png)
![image](https://user-images.githubusercontent.com/84871522/119712122-20464980-be15-11eb-8949-9d9999942582.png)

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

**Comment by tsanislavgatev** [comment]
Hello colleagues,

Can you please take a look? @SAP/ui5-webcomponents-topic-p 

Best Regards,
Tsanislav

**Comment by ilhan007** [comment]
Hello @Teju-SFSF which version of ui5 web component are you using?

**Comment by Teju-SFSF** [comment]
It is 0.31.6.

**Comment by ilhan007** [comment]
Hello @Teju-SFSF it will be released with 0.31.8 in the beginning of the next week.
